### DISCUSSION TITLE
aiortc doesn't work with flask.

### DISCUSSION THREAD

**Discussion by kevinjacb** [aiortc/aiortc#1049] [Q&A]
I modified both the server example and webcam.py example to work with Flask(just that no other modification) and it stopped working. ICE state changes from checking to disconnected after a few seconds. Seems like it works only with aiohttp.web or am I missing something? @jlaine Please help. Thank you. (I understand there is a similar discussion but there seems to be no response hence the new one.)

**Comment by MrJaysa** [comment]
try it out with asynchronous servers
